sprout/core/create_dirs.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+
+def create_dirs(path: list[Path]) -> list[Path]:
+    """Creates directories from a list of directories.
+
+    Args:
+        path: A list of paths pointing to where directories should be created.
+
+    Raises:
+        NoDirectoryError: If the directory wasn't created
```

Comment:
Should this be NotADirectoryError?
```suggestion
        NotADirectoryError: If the directory wasn't created
```

Reply:
I think it is short for 'no directory was created', so No is correct...?

Reply:
I was thinking more in the sense that if we think about when this function would raise an error, it would be if the input `path` is not a directory, right? 

`NotADirectoryError` is also a built-in exception, see the docs [here](https://docs.python.org/3/library/exceptions.html#NotADirectoryError), whereas the `NoDirectoryError` is a custom exception. And it might be nicer to use the built-in exceptions when it makes sense for us to do so :)